packages/base/src/3dview/mainview.tsx

```diff
@@ -1120,7 +1120,8 @@ export class MainView extends React.Component<IProps, IStates> {
 
       if (wireframeEnabled !== undefined) {
         this.setState({ wireframe: wireframeEnabled }, () => {
-          this._scene.traverse(child => {
+          // @ts-ignore
+          this._meshGroup.traverse(child => {
```

Comment:
Let's try our best to not use `@ts-ignore` in the codebase. I guess you had an TypeScript error here because `this._meshGroup` could potentially be undefined?

In that case I would do:
```
if (this._meshGroup !== undefined) {
  // Do the traverse
  // ...
}
```

Reply:
Thanks @martinRenou, done in [`ca807d3`](https://github.com/jupytercad/JupyterCAD/pull/390/commits/ca807d33c84b31d0dd6d765fc9354f3a35ba61f8)